keyboards/andean_condor/info.json

```diff
@@ -0,0 +1,87 @@
+                {"matrix": [0,4], "x":4, "y":0, "label":"R"},
+                {"matrix": [0,5], "x":5, "y":0, "label":"T"},
+                
+                {"matrix": [4,5], "x":10, "y":0, "label":"Y"},
+                {"matrix": [4,4], "x":11, "y":0, "label":"U"},
+                {"matrix": [4,3], "x":12, "y":0, "label":"I"},
+                {"matrix": [4,2], "x":13, "y":0, "label":"O"},
+                {"matrix": [4,1], "x":14, "y":0, "label":"P"},
+                {"matrix": [4,0], "x":15, "y":0, "label":"BS"},
+                
+                {"matrix": [1,0], "x":0, "y":1, "label":"CTRL"},
+                {"matrix": [1,1], "x":1, "y":1, "label":"A"},
+                {"matrix": [1,2], "x":2, "y":1, "label":"S"},
+                {"matrix": [1,3], "x":3, "y":1, "label":"D"},
+                {"matrix": [1,4], "x":4, "y":1, "label":"F"},
+                {"matrix": [1,5], "x":5, "y":1, "label":"G"},
+                
+                {"matrix": [5,5], "x":10, "y":1, "label":"H"},
+                {"matrix": [5,4], "x":11, "y":1, "label":"J"},
+                {"matrix": [5,3], "x":12, "y":1, "label":"K"},
+                {"matrix": [5,2], "x":13, "y":1, "label":"L"},
+                {"matrix": [5,1], "x":14, "y":1, "label":":"},
+                {"matrix": [5,0], "x":15, "y":1, "label":"RET"},
+                
+                {"matrix": [2,0], "x":0, "y":2, "label":"SHIFT"},
+                {"matrix": [2,1], "x":1, "y":2, "label":"Z"},
+                {"matrix": [2,2], "x":2, "y":2, "label":"X"},
+                {"matrix": [2,3], "x":3, "y":2, "label":"C"},
+                {"matrix": [2,4], "x":4, "y":2, "label":"V"},
+                {"matrix": [2,5], "x":5, "y":2, "label":"B"},
+                
+                {"matrix": [6,5], "x":10, "y":2, "label":"N"},
+                {"matrix": [6,4], "x":11, "y":2, "label":"M"},
+                {"matrix": [6,3], "x":12, "y":2, "label":","},
+                {"matrix": [6,2], "x":13, "y":2, "label":"."},
+                {"matrix": [6,1], "x":14, "y":2, "label":"/"},
+                {"matrix": [6,0], "x":15, "y":2, "label":";"},
+
+                {"matrix": [3,0], "x":3.5, "y":4, "label":"lb1"},
+                {"matrix": [3,1], "x":4.5, "y":4, "label":"lb2"},
+                {"matrix": [3,2], "x":5.5, "y":4, "label":"lb3"},
+                {"matrix": [3,3], "x":6.5, "y":4, "label":"lb4"},
+                {"matrix": [3,4], "x":6.5, "y":3, "label":"lt2"},
+                {"matrix": [3,5], "x":5.5, "y":3, "label":"lt1"},
+                
+                {"matrix": [7,5], "x":9.5, "y":3, "label":"rt2"},
+                {"matrix": [7,4], "x":8.5, "y":3, "label":"rt1"},
+                {"matrix": [7,3], "x":8.5, "y":4, "label":"rb3"},
+                {"matrix": [7,2], "x":9.5, "y":4, "label":"rb4"},
+                {"matrix": [7,1], "x":10.5, "y":4, "label":"rb5"},
+                {"matrix": [7,0], "x":11.5, "y":4, "label":"rb6"}
```

Comment:
> it would require re-ordering the info.json layout items

```suggestion
                {"matrix": [3,4], "x":6.5, "y":3, "label":"lt2"},
                {"matrix": [3,5], "x":5.5, "y":3, "label":"lt1"},

                {"matrix": [7,5], "x":9.5, "y":3, "label":"rt2"},
                {"matrix": [7,4], "x":8.5, "y":3, "label":"rt1"},

                {"matrix": [3,0], "x":3.5, "y":4, "label":"lb1"},
                {"matrix": [3,1], "x":4.5, "y":4, "label":"lb2"},
                {"matrix": [3,2], "x":5.5, "y":4, "label":"lb3"},
                {"matrix": [3,3], "x":6.5, "y":4, "label":"lb4"},

                {"matrix": [7,3], "x":8.5, "y":4, "label":"rb3"},
                {"matrix": [7,2], "x":9.5, "y":4, "label":"rb4"},
                {"matrix": [7,1], "x":10.5, "y":4, "label":"rb5"},
                {"matrix": [7,0], "x":11.5, "y":4, "label":"rb6"}
```